WcaOnRails/app/assets/stylesheets/links.scss

```diff
@@ -14,5 +14,6 @@ form[target=_blank] button[type=submit] {
 }
 
 :target {
-  background-color: yellow;
+  // TODO: why?
```

Comment:
So you can tell what I'm linking you to here: https://www.worldcubeassociation.org/regulations/#Z3